client/src/components/HomePage/modal/LoginModal/style.css

```diff
@@ -1,21 +1,19 @@
 .modal {
-  position: absolute;
-  z-index: 99;
-  top: 40%;
-  left: 50%;
-  width: 350px;
-  margin-top: -100px;
-  margin-left: -200px;
-  border-radius: 5px;
-  box-shadow: 0 0 5px #00000075;
-  padding: 5px 0px;
-  animation: fadeInOpacity 0.4s 1;
-  text-align: center;
-  background-color: white;
+  top: 0;
+  left: 0;
+  width:100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.6);
 }
 
 .modelContent {
```

Comment:
please use valid naming conventions. I suggest the BEM. But you can use other conventions if you all agree to them.

Reply:
http://getbem.com/naming/